src/instructlab/cli/model/train.py

```diff
@@ -245,6 +245,7 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     cls=clickext.ConfigOption,
     config_sections=ADDITIONAL_ARGUMENTS,
     type=click.BOOL,
+    required=True,
```

Comment:
could you add the comment saying ` # default from config` to keep it consistent with other additonal_args?